### DISCUSSION TITLE
Correct way to disable one LazyExtras dependency

### DISCUSSION THREAD

**Discussion by grifferz** [LazyVim/LazyVim#3742] [Q&A]
Hi, I like most of `lang.markdown` from LazyExtras but I don't like `headlines.nvim`. Should I be enabling `lang.markdown` and then adding something to disable `headlines.nvim`, or should I be manually enabling all the other dependencies of `lang.markdown`?

Specifically I am interested in linting and formatting.

**Comment by iamxiaojianzheng** [comment]
Look at this.

https://www.lazyvim.org/configuration/plugins#-disabling-plugins
